services/cron/tasks.go

```diff
@@ -147,11 +147,6 @@ func GetTask(name string) *Task {
 func RegisterTask(name string, config Config, fun func(context.Context, *user_model.User, Config) error) error {
 	log.Debug("Registering task: %s", name)
 
-	i18nKey := "admin.dashboard." + name
-	if _, ok := translation.TryTr("en-US", i18nKey); !ok {
-		return fmt.Errorf("translation is missing for task %q, please add translation for %q", name, i18nKey)
-	}
-
```

Comment:
If there is a strong reason to remove this check, it's fine to me.

But IMO I prefer to keep this check. Because the i18n key `admin.dashboard.xxx` is not obvious, it's easy to be forgotten (and it has been forgotten in history). This check will help developer to make sure that the i18n key won't be forgotten anymore if they are changing the task code.

Reply:
Oh pretty sure I removed it as it was using the weird TryTr function 😄. I will back it in using the appropriate function.

Reply:
Related PR:
* https://github.com/go-gitea/gitea/pull/18646

Reply:
https://github.com/go-gitea/gitea/pull/20096/commits/6d3e75484386ad30a4c877e8e27b88bd7c43de81